lib/danger/request_sources/github/github.rb

```diff
@@ -30,7 +30,8 @@ def initialize(ci_source, environment)
         self.support_tokenless_auth = false
         self.dismiss_out_of_range_messages = false
 
-        @token = @environment["DANGER_GITHUB_API_TOKEN"]
+        @access_token = @environment["DANGER_GITHUB_API_TOKEN"]
```

Comment:
I renamed this instance variable. 
  This value is concealed. so it may not affect.